gc/base/MarkingScheme.hpp

```diff
@@ -139,13 +141,7 @@ class MM_MarkingScheme : public MM_BaseVirtual
 	MMINLINE bool
 	inlineMarkObject(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, bool leafType=false)
 	{
-		bool didMark = false;
```

Comment:
Verbosity never hurts as source file size is never an issue.  I am ok with these changes as there are only 2 logical operations happening on the line.  In the future for changes like these can they be pulled into a separate PR / commit as they are just refactoring to functions that otherwise would not have been changed.  Thanks